test_projects/cnode/test/example_test.exs

```diff
@@ -8,6 +8,10 @@ defmodule ExampleTest do
     [cnode: cnode]
   end
 
+  test "nil", context do
+    assert nil == Unifex.CNode.call(context[:cnode], :test_nil, [:unifex])
```

Comment:
Last argument passed to `call` is list of arguments passed to native function, so it should be `[]` instead of `[:unifex]`